### DISCUSSION TITLE
子应用卸载后怎么释放内存？

### DISCUSSION THREAD

**Discussion by involker** [jd-opensource/micro-app#447] [General]
目前子应用卸载后内存没有被释放，反复加载几次后导致内存越来越大

**Comment by bailicangdu** [comment]
https://micro-zoe.github.io/micro-app/docs.html#/zh-cn/advanced?id=_2%e3%80%81%e6%80%a7%e8%83%bdamp%e5%86%85%e5%ad%98%e4%bc%98%e5%8c%96

**Reply by involker** [reply]
测试了下确实有效果，但是这个 #138 依然存在，有更好的方案吗
